asllib/Typing.ml

```diff
@@ -2780,7 +2886,7 @@ module Annotate (C : ANNOTATE_CONFIG) : S = struct
           | (_, ret_type) :: args -> (ret_type, List.map snd args)
         in
         let _, _, func_sig' =
-          try Fn.subprogram_for_name loc env func_sig.name arg_types
+          try Fn.subprogram_for_name loc env V0 func_sig.name arg_types
```

Comment:
Why specifically the `V0` version? If we pass an annotated `func` we can apply the correct version, no?